### DISCUSSION TITLE
Problem z wysyłką maili przez SMTP dla Azure SendGrid

### DISCUSSION THREAD

**Discussion by krupers** [AzureCommunityPL/group#34] [Pomoc] [answered]
Kochani mam pytanie odnośnie SendGrid z Azure...

Mam wszystko skonfigurowane, w większości skrypty o które pytałem na grupie Office już działają ale... no właśnie chciałem do systemu sklepowego dodać SMTP z SendGrid, aby sklep też wysyłał wiadomości korzystając z tej puli, lecz o ile mam adres, port, hasło (wygenerowane API), o tyle nie mam nazwy użytkownika.

Czy można jakość wyciągnąć ją? Bo jedyna jaką widzę to ten randomowy "email", który... nie jest przyjmowany w systemie ustawień poczty.

No i najgorsze - nie mogę tego zmienić, bo Azure według wszelkiej literatury nie wspiera tego zabiegu (auto-logowanie jest przez właśnie ten adres) ...

Ma ktoś pomysł co będzie nazwą użytkownika? (nie jest to sam przedrostek przed "@", ani nie jest to nazwa jaką nadałem przy tworzeniu usługi)
![Capture](https://user-images.githubusercontent.com/9113275/91313317-87b36800-e7b5-11ea-9ec1-3ea129ae2b0e.PNG)

**Comment by kaluzaaa** [comment, marked as answer]
Wejdź w panel i przeklikaj się przez wizard ;-)

![image](https://user-images.githubusercontent.com/4700951/91314645-14125a80-e7b7-11ea-8756-fe8559aec8bb.png)

**Reply by krupers** [reply]
A widzisz, tutaj nie szukałem! Dodałem nowy klucz i działa - wysyła! :)

Dzięki Łukaszu!
